src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -69,6 +69,13 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
+    def deleteCluster(googleId: UUID):DBIO[Int] = {
```

Comment:
Doesn't this do the same thing as `deleteByGoogleId` below?

Reply:
Nevermind! I see `deleteByGoogleId` hard-deletes, while this soft-deletes. We might want to remove the hard-deleting methods if we're not going to use them.

Reply:
We shouldn't even _have_ the hard-delete IMO.